components/accordion-menu/README.md

```diff
@@ -89,3 +91,44 @@ You can also add the following attributes to the gameface-accordion-panel
 |---|---|---|---|
 |disabled  | Boolean   |false   | If the panel is disabled. You can't expand or shrink disabled panels  |
 |expanded   | Boolean   |false   | If the panel is expanded on load    |
+
+### v3.1.0
+
+You can change the attributes mentioned above runtime via `setAttribute`, `removeAttribute` or with JavaScript.
+
+For example we can change the `multiple` of accordion menu like that
+```javascript
+const accordionMenu = document.querySelector('gameface-accordion-menu');
+// If we want to enable multiple of the accordion menu
+accordionMenu.setAttribute('multiple', ''); 
+// or
+accordionMenu.multiple = true;
+// And if we want to disable the multiple of the accordion menu
+accordionMenu.removeAttribute('multiple');
+// or
+accordionMenu.multiple = false;
+```
+
+**Not that when disabling the `multiple` functionality of the accordion-menu and there are more than one accordion-panels expanded then it will close all of them if are not disabled and leave just the first one expanded.**
```

Comment:
```suggestion
**Note that when disabling the `multiple` functionality of the accordion-menu and there are more than one accordion-panels expanded then it will close all of them if are not disabled and leave just the first one expanded.**
```

Reply:
Fixed